lib/core/include/irods_configuration_keywords.hpp

```diff
@@ -76,6 +76,7 @@ namespace irods
     extern const std::string CFG_ADDRESS_KW;
 
     // advanced settings
+    extern const std::string DELAY_RULE_EXECUTORS_KW;
```

Comment:
alpha order?  do we have a convention for these things?

Reply:
We don't.

Reply:
will create an issue about this file.